manager/api/grpc/server.go

```diff
@@ -60,3 +64,16 @@ func (s *grpcServer) Process(stream manager.ManagerService_ProcessServer) error
 	})
 	return eg.Wait()
 }
+
+func (s *grpcServer) Heartbeat(stream manager.ManagerService_HeartbeatServer) error {
+	p, ok := peer.FromContext(stream.Context())
+	if !ok {
+		return errors.New("failed to get peer from context")
+	}
+	for {
+		if _, err := stream.Recv(); err != nil {
```

Comment:
What is this that we are receiving? I thought that in the `Heartbeat ()` fnc we have just to send a heartbeat message.

Reply:
this on the server side where heartbeat requests are received